### PR TITLE
Wallets: add fee handling info to Choose Your Wallet page

### PR DESCRIPTION
**Pull request by harding**
Preview: http://dg0.dtrt.org/en/choose-your-wallet

This PR adds an extra entry to each wallet's listing describing how it handles fees.  For example, here's Bitcoin Core's updated entry:

This is really @crwatkins PR (although I support it); I just happened to have a Bitcoin.org build and hosting environment already setup for quick previewing and testing, so I'm opening the PR with his assent.

All fee-related text is at this point in the diff: https://github.com/harding/bitcoin.org/commit/61b28f582c2c308dc5f63a30a77006b6810476cd#diff-625d1a93b257504de31285773c0f35fd

Testing: I previewed each entry on Firefox and a few randomishly-selected entries on Chrome for Android.  More browser testing always welcome since this PR includes a couple CSS changes in a complicated layout.

### CODE REVIEW

**Comment by theymos** [comment]
Very good idea!

I don't think that Bitcoin Core supports CPFP except through really hacky raw transactions stuff.

**Comment by crwatkins** [comment]
@harding thanks so much for the build, preview, and CSS tweaks! 

Testing:  I previewed all entries on Safari and a few entries on Chrome on both an iOS phone and an iOS tablet.

@theymos I believe you are correct about CPFP and Core.  CPFP is only possible with many CLI commands with argument substitution.  However, Core supports RBF with a single simple CLI command (although no GUI) which allowed it to just squeak by with a "good" score on fees.

**Comment by crwatkins** [comment]
Thanks to @jlegoupil for the original issue https://github.com/bitcoin-dot-org/bitcoin.org/issues/1255 that introduced this idea.  In response to @instagibbs [note](https://github.com/bitcoin-dot-org/bitcoin.org/issues/1255#issuecomment-198924011) I changed the topic of the scoring from just "delay" to the more inclusive "fees".

**Comment by instagibbs** [comment]
awesome work (completely forgot my involvement)

**Comment by jonasschnelli** [comment]
There is a QT fee bumper on the way for 0.15: https://github.com/bitcoin/bitcoin/pull/9697
Also, CPFP is kinda-possible via GUI if you enable `-spendzeroconfchange` and use coin-control 🤓

**Comment by schildbach** [comment]
I think we should not rate "Full control" higher than the various forms of automatic. Full control also means complicated to use, not usable for non-technical people, etc. It's fine if there are power-user-wallets, but the others shouldn't be punished.

**Comment by crwatkins** [comment]
@jonasschnelli I stand corrected on your Core GUI coin-control CPFP wizardry!

**Comment by instagibbs** [comment]
@schildbach I don't see any wallets being punished for that, at least Coin.Space for example.

**Comment by schildbach** [comment]
@instagibbs checkgood is rated higher than checkpass.

File: _templates/choose-your-wallet.html [hunk 1]
```diff
@@ -105,6 +108,7 @@
           transparency: "checkpasstransparencyopensource"
           environment: "checkfailenvironmentdesktop"
           privacy: "checkgoodprivacyimproved"
+          fees: "checkpassfeecontroloverride"
```
Comment (achow101) [inline _templates/choose-your-wallet.html]: Armory's next version (which will be released soon) will contain CPFP and RBF. Is that good enough to have it listed as full control now or should I wait and PR that after the release?

**Comment by crwatkins** [comment]
@schildbach I certainly see your point, particularly for Core having to use a CLI option for RBF, however there are other wallets that make RBF/CPFP fairly simple, including your wallet!  Perhaps the "Full control" is poorly named?  It was named "Full" to be consistent with some of the other "top scores" in the other categories.  In this case, you'll see the complete text of "Full control" requires RBF or CPFP.  I can see that "Full control" might be taken as meaning the ability to set some arbitrary fee by the user (which is not what it means in this case).

**Comment by achow101** [comment]
Looking at the prevous, it seems that Coinapault, Coinbase, and Xapo have no rating. Is that intentional?

**Comment by crwatkins** [comment]
@achow101 Let us know as soon as it is released or just submit a PR then.   All of our scorings reflect released versions.  Thanks!

**Comment by schildbach** [comment]
@crwatkins Yes indeed I thought full control means being able to type a fee amount. So I'm sorry for my misunderstanding.

**Comment by crwatkins** [comment]
@schildbach Thanks!  You pointed out my poor choice of title.  I'll see if I can come up with something better.

Just as a note to meticulous reviewers:  There are two scores that are rated at the same "level" of "pass".  One involves providing dynamic fees and the other provides dynamic fees with the ability for the user to override them.

**Comment by crwatkins** [comment]
@achow101 That is mostly due to my laziness; I just noticed that this morning also.  There has been some re-org recently related to custodial wallets, [Bitcoin banks](https://github.com/bitcoin-dot-org/bitcoin.org/issues/1109) and [exchanges](https://github.com/bitcoin-dot-org/bitcoin.org/pull/1469) which may still be ongoing.  Until this morning, I thought [my recommendation](https://github.com/bitcoin-dot-org/bitcoin.org/issues/1109#issuecomment-268252161) to move them to the new category had already happened, but some of that still seems in progress.  In the meantime, I would be happy to see a PR to score them.

**Comment by wbnns** [comment]
Unless others object, this will be merged on Sunday, April 23rd.

**Pull request merged**